tensorvision/utils.py

```diff
@@ -8,12 +8,12 @@
 import logging
 import numpy as np
 import os
+import sys
 import time
 
 from datetime import datetime
 
 # https://github.com/tensorflow/tensorflow/issues/2034#issuecomment-220820070
```

Comment:
Ah, ok, I didn't see that it was imported twice. Yes, you are right. However, please keep the comment right above the numpy import, so that we know why it is there (even if numpy is not used).